.github/workflows/build.yml

```diff
@@ -55,6 +55,7 @@ jobs:
 
       - name: Commit unstaged readme/recompile changes if there are code changes to the action
         if: steps.action-code.outputs.HAS_CHANGES == 'true'
+        id: commit-changes
```

Comment:
I don't want to have one off changes like this in an action.  All of our javascript actions are built the same way so having one of them different doesn't work.

Reply:
Now that the ownership of this repo has shifted back to BC Swat team, you can ignore my request for input from Infra Purple.

And since the permissions have now been resolved, I don't need this change to assist me with these commit errors.